internal/run/convert.go

```diff
@@ -88,7 +146,27 @@ func (c *Convert) Execute() error {
 		Context: c.kubeContext,
 	}
 
-	return convertcmd.Convert(c.convertOptions, kc)
+	clientset, err := clientsetFromFile(c.kubeConfig)
+	if err != nil {
+		return err
+	}
+
+	configmaps, err := c.getV2ReleaseConfigmaps(clientset)
+	if err != nil {
+		return err
+	}
```

Comment:
**nit:** This is conditional on `if !c.convertOptions.DeleteRelease`

Reply:
duplicate how?, convert should run only once

Reply:
getV2ReleaseConfigmaps is just a read operation, it can also run when delete release is set, as the delete will happen in Convert

Reply:
Just pointing out that there is no need to retrieve releases if they are going to be deleted.